AdminWebsite/AdminWebsite/Startup.cs

```diff
@@ -123,5 +131,20 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseMiddleware<ExceptionMiddleware>();
         }
+
+        private static Func<TokenValidatedContext, Task> OnTokenValidated()
+        {
+            return async ctx =>
```

Comment:
Instead of returning a function here, just rewrite the function to be the delegate?

Reply:
Done